src/earn/EarnEnterAmount.tsx

```diff
@@ -454,6 +479,87 @@ function TransactionDetails({
   )
 }
 
+function FeeDetailsBottomSheet({
+  forwardedRef,
+  title,
+  testID,
+  feeCurrency,
+  estimatedFeeAmount,
+  maxFeeAmount,
+}: {
+  forwardedRef: React.RefObject<BottomSheetModalRefType>
+  title: string
+  testID: string
+  feeCurrency?: TokenBalance
+  estimatedFeeAmount?: BigNumber
+  maxFeeAmount?: BigNumber
+}) {
+  const { t } = useTranslation()
+
+  return (
+    <BottomSheet forwardedRef={forwardedRef} title={title} testId={testID}>
+      <View style={styles.bottomSheetTextContent}>
+        <View style={styles.gap8}>
+          <View style={styles.lineItem} testID="EstNetworkFee">
+            <Text style={styles.bottomSheetLineLabel}>
+              {t('earnFlow.enterAmount.feeBottomSheet.estNetworkFee')}
+            </Text>
+            {feeCurrency && estimatedFeeAmount && (
+              <Text style={styles.bottomSheetLineLabelText} testID="EstNetworkFee/Value">
+                <TokenDisplay
```

Comment:
In the designs it looks like this should have a ≈ before it

Reply:
Updated in 30a9aedcee50d92a9b36463578adb3b1e1d1eeb4.